rosidl_runtime_cpp/include/rosidl_runtime_cpp/traits.hpp

```diff
@@ -15,11 +15,138 @@
 #ifndef ROSIDL_RUNTIME_CPP__TRAITS_HPP_
 #define ROSIDL_RUNTIME_CPP__TRAITS_HPP_
 
+#include <codecvt>
+#include <string>
 #include <type_traits>
 
 namespace rosidl_generator_traits
 {
 
+void value_to_yaml(bool value, std::ostream & out)
```

Comment:
Should be made inline.
Same for the other functions here.

Reply:
See same comment above.